.circleci/config.yml

```diff
@@ -37,7 +37,10 @@ jobs:
           command: yarn build
       - run:
           name: Running tests
-          command: yarn test
+          command: yarn test --coverage
+      - run:
+          name: Send coverage report
+          command: yarn coverage
```

Comment:
You have to add the CODECOV_TOKEN as environment variable here

Reply:
I don't think this is necessary, upload tokens are mainly needed for private repositories. See what I found on codecov about the [uplaod token](https://docs.codecov.com/docs/codecov-uploader#upload-token).

![Screen Shot 2021-09-09 at 23 35 29](https://user-images.githubusercontent.com/32770340/132834770-4672f28c-dc2e-4131-b76e-d272b0fc1bae.png)